fabtests/benchmarks/benchmark_shared.c

```diff
@@ -79,6 +82,7 @@ void ft_benchmark_usage(void)
 	FT_PRINT_OPTS_USAGE("-v", "enables data_integrity checks");
 	FT_PRINT_OPTS_USAGE("-k", "force prefix mode");
 	FT_PRINT_OPTS_USAGE("-j", "maximum inject message size");
+	FT_PRINT_OPTS_USAGE("-X", "maximum untagged message size");
```

Comment:
I think using a general `-X` only for untagged message size is not desired. We can introduce a long option `--max-msg-size` for it.